prac_10/testing.py

```diff
@@ -41,28 +42,51 @@ def run_tests():
     car = Car()
     assert car._odometer == 0, "Car does not set odometer correctly"
 
-    # TODO: 2. write assert statements to show if Car sets the fuel correctly
+    # 2. write assert statements to show if Car sets the fuel correctly
     # Note that Car's __init__ function sets the fuel in one of two ways:
     # using the value passed in or the default
     # You should test both of these
     car = Car(fuel=10)
+    assert car.fuel == 10, "Fuel must be 10"
+    car = Car()
+    assert car.fuel == 0, "Fuel must be 0"
 
 
 run_tests()
 
-# TODO: 3. Uncomment the following line and run the doctests
+# 3. Uncomment the following line and run the doctests
 # (PyCharm may see your >>> doctest comments and run doctests anyway.)
-# doctest.testmod()
+doctest.testmod()
 
-# TODO: 4. Fix the failing is_long_word function
+# 4. Fix the failing is_long_word function
 # (Don't change the tests, change the function!)
 
-# TODO: 5. Write and test a function to format a phrase as a sentence,
+# 5. Write and test a function to format a phrase as a sentence,
 # starting with a capital and ending with a single full stop.
 # Important: start with a function header and just use pass as the body
 # then add doctests for 3 tests:
 #   'hello' -> 'Hello.'
 #   'It is an ex parrot.' -> 'It is an ex parrot.'
 # and one more that you decide is a useful test.
 # Run your doctests and watch the tests fail.
-# Then write the body of the function so that the tests pass.
\ No newline at end of file
+# Then write the body of the function so that the tests pass.
+
+def format_phrase_as_sentence(phrase):
```

Comment:
You could just make it add a "." to the end of the line. I also think that if a person ends with a full stop, this will remove the full stop. (unless I'm mistaken.)